src/amplitude-client.js

```diff
@@ -1468,6 +1525,28 @@ if (BUILD_COMPAT_2_0) {
   };
 }
 
+/**
```

Comment:
nit: since `logErrorsOnEvents` is private, should it be `_logErrorsOnEvents` for convention?

Reply:
good point, lemme fix that

Reply:
@jooohhn done